demisto_sdk/commands/validate/initializer.py

```diff
@@ -504,7 +508,7 @@ def git_paths_to_basecontent_set(
                 if obj:
                     obj.git_status = git_status
                     # Check if the file exists
-                    if git_status in (GitStatuses.MODIFIED, GitStatuses.RENAMED):
+                    if git_status in (GitStatuses.MODIFIED, GitStatuses.RENAMED, None):
```

Comment:
Let's do that for None only in cases where the file is a pack_metadata.json file.